content/teaching-curriculum/chapter1_high_standards

```diff
@@ -0,0 +1,32 @@
+---
+_db_id: 
+content_type: project
+flavours:
+- teaching
```

Comment:
misuse of flavour. 

If a project has the flavours python and node then it means that the project can either be done in python or node. You probably want them to submit markdown files? Mayb ethe flavour should be markdown